### ISSUE TITLE
Command proposal: set-layout-name

### ISSUE THREAD

**Issue by alex-courtis** [riverwm/river#1002]
Layout generators respond to a layout demand and commits with a layout name.

This may be fetched via the `layout_name` event and displayed on, say, a status bar.

This works well for the great majority of use cases except:
1. First starting river - the layout name is empty
2. Tag with no views is focused - the layout name is unchanged as no layout demand is made

Use case: layout generator tracks layout per-tag and updates the name.

1. Is not a big issue; the user knows their default layout.
2. The user is often shown the incorrect layout and unnecessarily changes to the actual layout. For layout generators that offer layout toggling it often results in the user toggling away from the actual layout.

**Comment by MaxVerevkin** [comment]
#974 addresses this, in a different way.

**Comment by ifreund** [comment]
Yeah, as discussed in #974 this should be done as part of the river-layout protocol if done at all. That PR doesn't seem to be going anywhere though so if someone wants to implement what I outlined in my comment there go for it.

**Comment by alex-courtis** [comment]
Thanks, protocol does seem quite sensible.

I'll take a look into it and attempt complete compatibility.
